src/App.js

```diff
@@ -0,0 +1,34 @@
+import { HashRouter as Router, Routes, Route } from 'react-router-dom';
+import Header from './components/header';
+import Books from './redux/books/books';
+import Categories from './redux/categories/categories';
+import './App.css';
+
```

Comment:
- [ ] Currently, the app is not showing anything. Kindly make sure you fix this error. 

![Capture](https://user-images.githubusercontent.com/64173189/149834514-bdb94c10-d165-40c9-b314-30d7a8f4f7d9.JPG)


😄 🛠️